tests/storages_tests/rdb_tests/test_storage.py

```diff
@@ -177,6 +183,71 @@ def test_upgrade() -> None:
     assert old_version == new_version
 
 
+@pytest.mark.parametrize("optuna_version", ["0.9.0", "1.2.0", "1.3.0", "2.4.0", "2.6.0"])
+def test_upgrade(optuna_version: str) -> None:
+    def objective(trial: Trial) -> float:
+        x = trial.suggest_uniform("x", -5, 5)  # optuna==0.9.0 does not have suggest_float.
+        y = trial.suggest_int("y", 0, 10)
+        z = cast(float, trial.suggest_categorical("z", [-5, 0, 5]))
+        return x ** 2 + y ** 2 + z ** 2
+
+    def mo_objective(trial: Trial) -> Tuple[float, float]:
+        x = trial.suggest_float("x", -5, 5)
+        y = trial.suggest_int("y", 0, 10)
+        z = cast(float, trial.suggest_categorical("z", [-5, 0, 5]))
+        return x, x ** 2 + y ** 2 + z ** 2
```

Comment:
How about leaving a comment that these objective functions must be similar to those in `create-db.py`?

Reply:
Thank you for your suggestion. I reused these objective functions in the test_storage.py in https://github.com/optuna/optuna/pull/2890/commits/34d8ef962981ce947bb2c45406ef170b8bf76183. I think we can reduce the maintenance cost, but what do you think?

Reply:
Oh, that's even better. 👍